frontend/lib/src/components/widgets/Checkbox/Checkbox.tsx

```diff
@@ -241,8 +243,9 @@ class Checkbox extends React.PureComponent<Props, State> {
                   outline: 0,
                   width: "1rem",
                   height: "1rem",
-                  marginTop: "0.30rem",
+                  marginTop: "0.25rem",
                   marginLeft: 0,
+                  marginBottom: 0,
```

Comment:
nit: could this be a styledCheckmark instead of inline css? if yes, it might make sense to take the opportunity and do the refactoring here and unify our system, what do you think?

Reply:
Unfortunatly not. These stylings for baseweb sub-components are not compatible with styled-components :( you can see the same in most of the other baseweb components.